spec/dummy/config/boot.rb

```diff
@@ -0,0 +1 @@
+require 'bundler/setup'
```

Comment:
some of the stuff in dummy is so I could get bin/rails commands working since I was losing my mind